2.feature_extraction/0.create_loaddata_csvs.ipynb

```diff
@@ -418,13 +418,13 @@
        "3     /media/18tbdrive/ALSF_pilot_data/SN0313537/BR0...              False  \n",
        "4     /media/18tbdrive/ALSF_pilot_data/SN0313537/BR0...              False  \n",
        "...                                                 ...                ...  \n",
-       "2842  /media/18tbdrive/ALSF_pilot_data/SN0313537/202...               True  \n",
-       "2843  /media/18tbdrive/ALSF_pilot_data/SN0313537/202...               True  \n",
-       "2844  /media/18tbdrive/ALSF_pilot_data/SN0313537/202...               True  \n",
-       "2845  /media/18tbdrive/ALSF_pilot_data/SN0313537/202...               True  \n",
-       "2846  /media/18tbdrive/ALSF_pilot_data/SN0313537/202...               True  \n",
+       "5002  /media/18tbdrive/ALSF_pilot_data/SN0313537/202...               True  \n",
+       "5003  /media/18tbdrive/ALSF_pilot_data/SN0313537/202...               True  \n",
+       "5004  /media/18tbdrive/ALSF_pilot_data/SN0313537/202...               True  \n",
+       "5005  /media/18tbdrive/ALSF_pilot_data/SN0313537/202...               True  \n",
+       "5006  /media/18tbdrive/ALSF_pilot_data/SN0313537/202...               True  \n",
```

Comment:
Same comment as above.

Reply:
I am not sure if I can consider it expected, but the data never modified after running so I am assuming this is a minor artifact from the notebook.